approved_tags.yaml

```diff
@@ -0,0 +1,103 @@
+---
+- "accessibilità"
+- "ia"
+- "android"
+- "api"
+- "apl"
+- "arte"
+- "domanda"
+- "assembly"
+- "audio"
+- "libro"
+- "browsers"
+- "c"
+- "c++"
+- "clojure"
+- "scienzecognitive"
+- "compilatori"
+- "informaticateorica"
+- "crittografia"
+- "criptovalute"
+- "css"
+- "cultura"
+- "d"
+- "database"
+- "debugging"
+- "design"
+- "devops"
+- "sistemidistribuiti"
+- "dotnet"
+- "dragonflybsd"
+- "istruzione"
+- "elixir"
+- "elm"
+- "emacs"
+- "erlang"
+- "evento"
+- "metodiformali"
+- "fortran"
+- "freebsd"
+- "giochi"
+- "go"
+- "grafica"
+- "hardware"
+- "haskell"
+- "storia"
+- "illumos"
+- "ios"
+- "java"
+- "javascript"
+- "lavoro"
+- "legge"
```

Comment:
questo tradotto così in italiano è ambiguo. Dovremmo trovare un sinonimo

Reply:
Normativa?

Reply:
Legislazione? Aspetti Legali?

Reply:
Giurisprudenza? Oppure mi piace legislazione come consigliato da @karjudev

Reply:
Io concordo con @karjudev .

Reply:
Corretto con `"legislazione"`